DOCS-14166: Update SDS setup landing page (#36587)
DataDog/documentation: content/en/security/sensitive_data_scanner/setup/_index.md

---
title: Sensitive Data Scanner Setup
description: Set up Sensitive Data Scanner to detect sensitive content in your telemetry data, LLM Observability data, cloud storage, and code repositories.
disable_toc: false
aliases:
  - /sensitive_data_scanner/setup
further_reading:
    - link: "/security/sensitive_data_scanner/"
      tag: "Documentation"
      text: "Sensitive Data Scanner"
    - link: "/security/sensitive_data_scanner/scanning_rules/"
      tag: "Documentation"
      text: "Learn more about scanning rules"
    - link: "/security/sensitive_data_scanner/guide/investigate_sensitive_data_findings/"
      tag: "Documentation"
      text: "Investigate sensitive data findings"
    - link: "/security/sensitive_data_scanner/guide/create-monitors-for-sensitive-data/"
      tag: "Documentation"
      text: "Create monitors for sensitive data"
---

## Overview

> Set up Sensitive Data Scanner based on the data source you want to scan. Each source uses its own setup process, so you only need to configure the sources relevant to your needs.

Reviewer: ```suggestion
Set up Sensitive Data Scanner for each data source you want to scan. Each source uses its own setup process, so you only need to configure the sources relevant to your needs.
```
I was trying to figure out what "based on" meant (for example, maybe you'd only have to do it once, but the _method_ would change depending on the data sources you had?). I think this might be a bit more straightforward.
